src/services/firebase.tsx

```diff
@@ -2,4 +2,26 @@ import firebase from 'firebase';
 import { firebaseConfig } from '../config';
 
 const fire = firebase.initializeApp(firebaseConfig);
+const fireAuth = fire.auth();
+
+export const fireApi = {
```

Comment:
i think that this should be default export, because we dont use the 'fire' anywhere, and call its methods through our fireApi interface